src/Stryker.Core/Stryker.Core.UnitTest/Mutators/NegateConditionMutatorTests.cs

```diff
@@ -61,19 +60,5 @@ public void MutatesStatementWithMethodCallWithNoArguments(string method)
             mutation.ReplacementNode.ToString().ShouldBe("!(Method())");
             mutation.DisplayName.ShouldBe("Negate expression");
         }
-
-        [Theory]
-        [InlineData("var y = x is object result ? result.ToString() : null;")] // can't mutate inline var declaration
```

Comment:
I checked: the new mutator mutated the condition into `true` or `false`, meaning it could not resolve `result`.

I now made sure the mutator does not mutate conditions that contain a declaration pattern.